spec/factories/education_benefits_claims.rb

```diff
@@ -7,11 +7,17 @@
     end
   end
 
-  %w[1990e 1990n].each do |form_type|
+  %w[1990e 1990n 1995s].each do |form_type|
     factory :"education_benefits_claim_#{form_type}", class: 'EducationBenefitsClaim' do
```

Comment:
Before this gets even more opaque, let's break these out into three factories. 

It seems weird that they all inherit from the same class but they're different factories. Are there really underlying traits here `education_benefits_claim trait: form_1990e`? Building factory *definitions* programmatically makes it harder to lint, let alone understand.

Reply:
Sounds good!